tests/test_app.py

```diff
@@ -78,6 +85,22 @@ def test_whole_audit_flow(client):
     assert result2["riskLimit"] == 5
 
 
+def test_reset_election(client):
```

Comment:
I was under the impression that the reset button was only part of the single-jurisdiction flow, so should only work with anonymous users.

Reply:
Ah. I suppose you're probably right. In that case… should I just decline this or require that the user be anonymous?

Reply:
(Aside: if we do want to keep single vs multi as separate flows, I think we should heavily segregate them in the backend code.)

Reply:
What do you mean by "decline" vs "require user be anonymous"? I think this could just return a Forbidden or Unauthorized, no strong preference

I agree on segregating the flows. That's why I pushed for a separate API. I don't really want to keep both but I don't understand the product context well enough to know what makes sense

Reply:
I meant "decline this PR" 😉  I think I'll do that for now.